src/app/components/option-item/option-item.scss

```diff
@@ -42,43 +42,3 @@
   line-height: 28px;
   text-align: left;
 }
-
```

Comment:
We have some issue with this element (you can see white colored arrow)
![Screenshot 2024-05-13 at 12 08 10](https://github.com/rolling-scopes/site/assets/56927722/0f4a5f96-0c05-4685-bffc-c3b15cf82f78)

Reply:
Fixed